test/infrastructure/inmemory/pkg/server/api/watch.go

```diff
@@ -139,6 +198,16 @@ func (m *WatchEventDispatcher) Run(ctx context.Context, timeout string, w http.R
 	}
 	w.Header().Set("Transfer-Encoding", "chunked")
 	w.WriteHeader(http.StatusOK)
```

Comment:
Is it okay that we don't flush after this like before?

Reply:
IMHO more efficient to

* write header
* write objects
* flush

Instead of

* write header
* flush
* write objects
* flush

Reply:
I'm also okay with the second one though.

Reply:
Let's keep it as is (as it's working). I was just wondering :D